catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/CatalogCacheSource.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.cache;
+
+import ddf.catalog.source.Source;
+
+/**
+ * Used by {@link ddf.catalog.plugin.PostFederatedQueryPlugin} and
+ * {@link ddf.catalog.plugin.PreFederatedQueryPlugin} to determine
+ * if a source is the cache.
+ */
+public interface CatalogCacheSource extends Source {
```

Comment:
Since we expect federation plugins to use this interface, I could see it moving to `catalog-core-api` but we should mark as experimental for now.